tests/test_worker.py

```diff
@@ -52,6 +52,9 @@ async def process(self, input_message: InputMessage):
         assert len(messages) > 2
         assert len(messages) == len(output_config.results), len(output_config.results)
 
+        messages.sort(key=lambda x: x.request_id)
+        output_config.results.sort(key=lambda x: x[-1])
```

Comment:
Probably we should be using NamedTuple here

Reply:
Yes, will move in the next PR.